api/templates/client.mustache

```diff
@@ -294,6 +294,11 @@ func (c *APIClient) prepareRequest(
 		return nil, err
 	}
 
+	// Prepend request base path, if applicable
+    if c.cfg.Path != "" {
```

Comment:
I can't `make openai` locally so perhaps we still need some formatting here. But the code functions are proper.

```
./etc/generate-openapi.sh
WARNING: The requested image's platform (linux/amd64) does not match the detected host platform (linux/arm64/v8) and no specific platform was requested
Error opening file "/api/contract/openapi/src/legacy/paths/query.yml" 
ENOENT: no such file or directory, open '/api/contract/openapi/src/legacy/paths/query.yml'
make: *** [openapi] Error 1
```